cmd/thanos/receive.go

```diff
@@ -71,7 +71,8 @@ func registerReceive(m map[string]setupFunc, app *kingpin.Application) {
 
 	replicationFactor := cmd.Flag("receive.replication-factor", "How many times to replicate incoming write requests.").Default("1").Uint64()
 
-	tsdbBlockDuration := modelDuration(cmd.Flag("tsdb.block-duration", "Duration for local TSDB blocks").Default("2h").Hidden())
```

Comment:
ah wait, this should still be here

Reply:
yeah I added something from another commit on accident :stuck_out_tongue: 
fix coming in a sec :smile: